pkg/backend/watch.go

```diff
@@ -67,9 +67,19 @@ func Watch(ctx context.Context, b Backend, stack Stack, op UpdateOperation, appl
 	}()
 
 	events := make(chan notify.EventInfo, 1)
-	if err := notify.Watch(path.Join(op.Root, "..."), events, notify.All); err != nil {
-		return result.FromError(err)
+
+	if len(op.Opts.WatchPaths) > 0 {
+		for _, element := range op.Opts.WatchPaths {
+			if err := notify.Watch(path.Join(op.Root, element, "..."), events, notify.All); err != nil {
+				return result.FromError(err)
+			}
+		}
+	} else {
+		if err := notify.Watch(path.Join(op.Root, "..."), events, notify.All); err != nil {
+			return result.FromError(err)
+		}
 	}
+
```

Comment:
We could probably unify these two branches, and just have the default value (either provided here, or by the CLI code) be `[]string{""}`, so that the code in the first path works in all cases.

Reply:
Sounds good!

Reply:
I added the `[]string{""}` you proposed and unified the mentioned branching.